RNAwesomeCardIO.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name             = "RNAwesomeCardIO"
+  s.version          = package["version"]
+  s.summary          = package["description"]
+  s.requires_arc     = true
+  s.license          = package["license"]
+  s.homepage         = 'n/a'
+  s.authors          = { package["author"] => package["author"] }
```

Comment:
This will end up in a strange structure. Did you mean something like:
```
s.authors          = { package["author"]["name"] => package["author"]["email"] }
```

Reply:
Not sure about is this working? Doesn't have name and email separately keys, generally, people do hardcoded as https://github.com/invertase/react-native-firebase/blob/master/ios/RNFirebase.podspec